modules/nf-core/windowmasker/main.nf

```diff
@@ -0,0 +1,44 @@
+process WINDOWMASKER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::blast=2.13.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/blast:2.13.0--hf3cf87c_0':
+        'quay.io/biocontainers/blast:2.13.0--hf3cf87c_0' }"
+
+    input:
+    tuple val(meta), path(ref)
+
+    output:
+    tuple val(meta), path("${output}")  , emit: wm_intervals
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args1   = task.ext.args1    ?: ""
+    def args2   = task.ext.args2    ?: ""
+    def prefix  = task.ext.prefix   ?: "${meta.id}"
+    def outfmt  = task.ext.outfmt   ?: "interval"
+    output  = "${prefix}.${outfmt}"
+
+    """
```

Comment:
Can you add the `-mem` option to your commands? (You can access the specified memory like this: `task.memory.toMega()`

Reply:
Would this be as simple as ` def memory  = task.memory.toMega()`?

Reply:
that should work, in GATK modules they also add a check to see if `task.memory` is set. That could also be useful here I think